src/scenes/Home/index.tsx

```diff
@@ -61,17 +68,32 @@ export function Home() {
     localStorage.setItem(key, JSON.stringify(newTodos));
   };
 
+  const remainingTask = todos.filter((todo) => !todo.completed).length;
```

Comment:
Take a look at the hook `useMemo` for this: https://reactjs.org/docs/hooks-reference.html#usememo

We can go back to this on Wednesday